packages/lambdas/src/fhir-to-medical-record.ts

```diff
@@ -214,3 +216,19 @@ async function getCxsWithADHDFeatureFlagValue(): Promise<string[]> {
 
   return [];
 }
+
+function shouldSendMrSummaryBasedOnContents(html: string): boolean {
+  let atLeastOneSectionHasContents = false;
+
+  const dom = new JSDOM(html);
+  const document = dom.window.document;
+  const sections = document.querySelectorAll("div.section");
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  sections.forEach((section: any) => {
+    const th = section.querySelector("th");
+    if (th) atLeastOneSectionHasContents = true;
```

Comment:
Shouldn't we look for at least one row? Not a header?

Also, why don't we break out of the loop as soon as we find one instead of looping through everything?